lib/mountain_view/component.rb

```diff
@@ -19,9 +29,27 @@ def stubs_file
     end
 
     def stubs?
-      styleguide_stubs.is_a?(Array) && styleguide_stubs.any?
-    rescue Errno::ENOENT
-      false
+      styleguide_stubs.any?
+    end
+
+    def stubs_extra_info?
+      !stubs_extra_info.empty?
+    end
+
+    def stubs_extra_info
+      if styleguide_stubs.is_a?(Hash)
+        if styleguide_stubs.key?(:meta)
+          styleguide_stubs[:meta]
+        else
+          {}
+        end
+      else
+        {}
+      end
+    end
+
+    def stubs_correct_format?
```

Comment:
@GustavoCaso @kitop what do you guys think if we accept both an Array or a Hash so we can make it backwards compatible.

So if we receive an array we loop the array as we did before and if it's a Hash we check it has the key `:stubs`